package.json

```diff
@@ -15,7 +15,7 @@
     "fuzzing": "docker run -it --rm -v $PWD:/src trailofbits/eth-security-toolbox"
   },
   "devDependencies": {
-    "@chainlink/contracts": "0.5.1",
+    "@chainlink/contracts": "^1.1.0-beta.0",
```

Comment:
I'm thinking in the next iteration of the PR we can just use 1.1.0 (https://www.npmjs.com/package/@chainlink/contracts/v/1.1.0-beta.0?activeTab=versions) so we can merge that directly. Please bump this package.json version to 2.1.1 too.  and to make sure the package-lock.json updates correct, best to do npm uninstall @chainlink/contracts and then do npm i -D @chainlink/contracts@1.1.0 (....i think!)

Reply:
Left with -beta.0, since 1.1.0 doesn't exist quite yet, but updated version to 2.1.1.